src/hooks/auth.ts

```diff
@@ -49,10 +50,11 @@ export const useAuth = ({ middleware }: { middleware?: any } = {}) => {
     await api
       .post('/api/login', props)
       .then(async (resp) => {
-        {
-          console.log(resp.data, '--> resp.data');
-        }
         setAuthToken(resp.data);
+        setCookie(null, 'authToken', resp.data, {
+          maxAge: 30 * 24 * 60 * 60,
+          path: '/',
```

Comment:
what does `path` mean here?

Reply:
This `path` parameter here which means the cookie is valid for all paths on the server.